### PR TITLE
Add a local tilt environment without Pro Connect

### PR DESCRIPTION
**Pull request by rouja**

### CODE REVIEW

File: src/helm/helmfile.yaml [hunk 1]
```diff
@@ -45,9 +50,50 @@ releases:
...
+    missingFileHandler: Warn
+    namespace: {{ .Namespace }}
+    chart: bitnami/keycloak
+    version: 17.3.6
+    values:
+      - postgresql:
+          auth:
+            username: keycloak
+            password: keycloak
+            databaese: keycloak
```
Comment (qbey) [inline src/helm/helmfile.yaml]:
```suggestion
            database: keycloak
```

File: Makefile [hunk 2]
```diff
@@ -305,3 +305,6 @@ start-tilt: ## start the kubernetes cluster using kind
 	tilt up -f ./bin/Tiltfile
 .PHONY: build-k8s-cluster
 
+start-tilt-keycloak: ## start the kubernetes cluster using kind
```
Comment (lebaudantoine) [inline Makefile:308]: nit: precise the comment

Comment (lebaudantoine) [inline Makefile]: nit: precise the comment

File: src/backend/meet/settings.py [hunk 3]
```diff
@@ -329,6 +329,9 @@ class Base(Configuration):
     OIDC_VERIFY_SSL = values.BooleanValue(
         default=True, environ_name="OIDC_VERIFY_SSL", environ_prefix=None
     )
+    OIDC_VERIFY_SSL = values.BooleanValue(
+        default=True, environ_name="OIDC_VERIFY_SSL", environ_prefix=None
+    )
```
Comment (lebaudantoine) [inline src/backend/meet/settings.py]: should be removed by a rebase

File: src/helm/env.d/dev-keycloak/values.meet.yaml.gotmpl [hunk 4]
```diff
@@ -0,0 +1,188 @@
...
+  replicas: 1
+  envVars:
+    APP_NAME: summary-microservice
+    APP_API_TOKEN: password
+    AWS_STORAGE_BUCKET_NAME: meet-media-storage
+    AWS_S3_ENDPOINT_URL: minio.meet.svc.cluster.local:9000
+    AWS_S3_ACCESS_KEY_ID: meet
+    AWS_S3_SECRET_ACCESS_KEY: password
+    OPENAI_API_KEY: password
+    OPENAI_BASE_URL: https://albertine.beta.numerique.gouv.fr:32222/v1
```
Comment (lebaudantoine) [inline src/helm/env.d/dev-keycloak/values.meet.yaml.gotmpl]: wrong url

File: src/helm/env.d/dev-keycloak/values.meet.yaml.gotmpl [hunk 5]
```diff
@@ -0,0 +1,188 @@
...
```
Comment (lebaudantoine) [inline src/helm/env.d/dev-keycloak/values.meet.yaml.gotmpl]: same

File: src/helm/helmfile.yaml [hunk 6]
```diff
@@ -45,9 +50,50 @@ releases:
...
+          kind: ConfigMap
+          metadata:
+            name: meet-keycloak
+          data:
+            meet.json: |
+{{ readFile "../../docker/auth/realm.json" | replace "http://localhost:3200" "https://meet.127.0.0.1.nip.io" | indent 14 }}
+
   - name: minio
-    installed: {{ eq .Environment.Name "dev" | toYaml }}
+    installed: {{ or (eq .Environment.Name "dev") (eq .Environment.Name "dev-keycloak") | toYaml }}
```
Comment (lebaudantoine) [inline src/helm/helmfile.yaml]: would smth as: 
```
{{ contains .Environment.Name "dev" | toYaml }}
```
I've not tested it

File: src/helm/helmfile.yaml [hunk 7]
```diff
@@ -32,7 +36,8 @@ repositories:
 
 releases:
   - name: postgres
-    installed: {{ eq .Environment.Name "dev" | toYaml }}
+    installed: {{ or (eq .Environment.Name "dev") (eq .Environment.Name "dev-keycloak") | toYaml }}
+    missingFileHandler: Warn
```
Comment (lebaudantoine) [inline src/helm/helmfile.yaml:40]: Out of curiosity, why do we need to add `missingFileHandler` ?

File: src/helm/helmfile.yaml (hunk 6 above)
Comment (lebaudantoine) [reply src/helm/helmfile.yaml]: could we even do a yaml function, with an explicit name, `isDevEnvironment` ? wdyt ?

**Comment by lebaudantoine** [comment]
I'ven't tested it in local, will do before merging.

File: src/helm/helmfile.yaml (hunk 7 above)
Comment (rouja) [reply src/helm/helmfile.yaml:40]: It's because this file env.d/{{ .Environment.Name }}/values.secrets.yaml exist only for the new environment

**Comment by lebaudantoine** [comment]
Opening issue #281. Not blocking merging.

**Pull request merged**